tensorflow_datasets/core/features/image_feature.py

```diff
@@ -112,33 +112,36 @@ def __init__(self, shape=None, dtype=None, encoding_format=None):
     self._dtype = None
 
     # Set and validate values
-    self.set_encoding_format(encoding_format or 'png')
+    self.set_encoding_format(encoding_format)
     self.set_shape(shape or (None, None, 3))
     self.set_dtype(dtype or tf.uint8)
 
   def set_dtype(self, dtype):
     """Update the dtype."""
     dtype = tf.as_dtype(dtype)
-    acceptable_dtypes = ACCEPTABLE_DTYPES[self._encoding_format]
-    if dtype not in acceptable_dtypes:
-      raise ValueError('Acceptable `dtype` for %s: %s (was %s)' % (
-          self._encoding_format, acceptable_dtypes, dtype))
+    if self._encoding_format:
```

Comment:
To avoid extra indentation: 
```
acceptable_dtypes = ACCEPTABLE_DTYPES.get(self._encoding_format, None)
if acceptable_dtypes and dtype not in acceptable_dtypes:
      raise ValueError('Acceptable `dtype` for %s: %s (was %s)' % (
          self._encoding_format, acceptable_dtypes, dtype))
```